Automattic/wp-calypso
# Third-Party Themes: Add the product query to theme showcase

#### Proposed Changes

* Add the product query to the theme showcase so we can have the products available. This will allow us to find the product slug we should use to add the marketplace theme to the cart.
https://github.com/Automattic/wp-calypso/blob/56293b8c9f15ff2e887e18a40ca19b94bcf04785/client/state/themes/actions/add-external-managed-theme-to-cart.tsx#L74-L85
* The marketplace themes can have different billing cycles. Meaning that some themes can have monthly and yearly billing cycles, but some can have only monthly or yearly. We use the billing slug to find all the products that represent the different billing cycles for the same theme.
* This branch depends on #69616 - merge after the 

#### Testing Instructions

* On a free site, navigate to the theme showcase page.
* Find Makoney and click on the kebab menu. 
* Then you will see the option to "Upgrade to subscribe"; click on it.
* It should take you to checkout with the Business plan and the theme subscription added.
* You can also test this flow on a Business site and expect only to add the theme subscription to the cart.

Closes #69470
